core/src/main/scala/cats/data/NonEmptyVector.scala

```diff
@@ -405,10 +403,10 @@ sealed abstract private[data] class NonEmptyVectorInstances extends NonEmptyVect
    *
    * Also see the discussion: PR #3541 and issue #3069.
    */
-  implicit val catsDataInstancesForNonEmptyVectorBinCompat1: NonEmptyAlternative[NonEmptyVector]
-    with Bimonad[NonEmptyVector]
-    with NonEmptyTraverse[NonEmptyVector]
-    with Align[NonEmptyVector] =
+  implicit val catsDataInstancesForNonEmptyVectorBinCompat1
+    : NonEmptyAlternative[NonEmptyVector] & Bimonad[NonEmptyVector] & NonEmptyTraverse[NonEmptyVector] & Align[
+      NonEmptyVector
+    ] =
```

Comment:
Just for a record: this auto-formatting is hurting my eyes. I wonder if scalafmt can be configured to do better in such cases.